workflows/phageannotator/tests/run_cobra.nf.test

```diff
@@ -0,0 +1,85 @@
+    tag "CAT_CAT"
+    tag "BOWTIE2_BUILD"
+    tag "GENOMAD_ENDTOEND"
+    tag "GUNZIP"
+    tag "BACPHLIP"
+    tag "FASTQ_ALIGN_BOWTIE2"
+
+
+    test("Parameters: run_cobra = true") {
+        when {
+            workflow {
+                """
+                input[0] = Channel.of(
+                    [
+                        [ id:'test' ],
+                        [
+                            file(params.modules_testdata_base_path + 'genomics/prokaryotes/bacteroides_fragilis/illumina/fastq/test1_1.fastq.gz', checkIfExists:true),
+                            file(params.modules_testdata_base_path + 'genomics/prokaryotes/bacteroides_fragilis/illumina/fastq/test1_2.fastq.gz', checkIfExists: true)
+                        ]
+                    ],
+                    [
+                        [ id:'test2' ],
+                        [
+                            file(params.modules_testdata_base_path + 'genomics/prokaryotes/bacteroides_fragilis/illumina/fastq/test1_1.fastq.gz', checkIfExists:true),
+                            file(params.modules_testdata_base_path + 'genomics/prokaryotes/bacteroides_fragilis/illumina/fastq/test1_2.fastq.gz', checkIfExists: true)
+                        ]
+                    ]
+                )
+                input[1] = Channel.of(
+                    [
+                        [ id:'test' ],
+                        file(params.modules_testdata_base_path + 'genomics/prokaryotes/bacteroides_fragilis/illumina/fasta/test1.contigs.fa.gz', checkIfExists:true)
+                    ],
+                    [
+                        [ id:'test2' ],
+                        file(params.modules_testdata_base_path + 'genomics/prokaryotes/bacteroides_fragilis/illumina/fasta/test1.contigs.fa.gz', checkIfExists:true)
+                    ]
+                )
+                """
+            }
+            params {
+                outdir                  = "$outputDir"
+                run_cobra               = true
+                cobra_assembler         = "metaspades"
+                cobra_mink              = 27
+                cobra_maxk              = 127
+                // parameters to decrease sensitivity for test data
+                mash_screen_min_score   = 0.01
+                genomad_min_score       = 0.01
+                genomad_max_fdr         = 1
+                // speed up options since tools are fully tested in subworkflows
+                genomad_disable_nn      = true
+                genomad_sensitivity     = 0.1
+                checkv_minimal_db       = true
+            }
+        }
+
+        then {
+            assertAll(
+                { assert workflow.success },
+                { assert snapshot(workflow.out).match() },
+                { assert !path("${outputDir}/VirusLifestyle/bacphlip/test.filtered.fna.bacphlip").exists() },
+                { assert path("${outputDir}/VirusAbundance/coverm/contig/all_samples_alignment_results.tsv").readLines().size() == 4 }
```

Comment:
You don't want to check the extended contigs themselves? Or is that handled with teh `owkrflow.out`?

Reply:
Good call, I will add a check for the extended contigs as well 👍